conan/cache/cache.py

```diff
@@ -53,7 +54,9 @@ def _get_tmp_path():
 
     @staticmethod
     def _get_path(ref: ConanReference):
-        return md5(ref.full_reference)
+        value = ref.full_reference.encode("utf-8")
+        sha = sha256(value)
```

Comment:
BTW, a common trick to reduce from 2->1,3 ratio characters is to encode the hash from hexadecimal to base64

Reply:
Nice idea